python/cudf/cudf/core/series.py

```diff
@@ -6448,6 +6448,30 @@ def is_month_start(self):
         """
         return (self.day == 1).fillna(False)
 
+    @property
+    def is_year_start(self):
+        """
+        Boolean indicator if the date is the first day of the month.
```

Comment:
typo fix: i think you meant if the date is the first day of year

Reply:
Thanks for this catch!